### ISSUE TITLE
[Discussion] What would an orbiting base look like?

### ISSUE THREAD

**Issue by kd7uiy** [TeamPorcupine/ProjectPorcupine#1078]
Resource generation for an orbiting base is pretty slim. It might occasionally run across scrap satellites for scrap metal, but otherwise wouldn't have many raw resources. Some thoughts on how a space station would justify it's existence:

* Science. It could collect astronomy images, observe the planets and moons nearby, etc.
* Expedition- It could send expeditions to collect raw resources.
* Repair- It would make a good repair station, although spare parts might be hard to come by.
* Manufacturing- There might be some goods that can only be produced in Zero G, or are better if produced in Zero G. 
* Military- It could serve as a platform for missiles, lasers, etc.
* Communication- Facilitate communication, both with deep space objects as well as point to point.
* Warehouse- It might be easier to send goods from the planet to the space station, to be picked up by other ships at a profit.

Difficulties for an orbiting space station

* Where to get the resources? They likely require an infusion of outside goods.
* Day/Night cycles, they will have more temperature fluctuation and power fluctuation than in deep space, but not as much as on a planet. They will also have solar power faults when in eclipse, and require batteries or shutting down some subsystems, or at least an alternative mean to generate power when in eclipse.

**Comment by mikejbrown** [comment]
> Warehouse- It might be easier to send goods from the planet to the space station, to be picked up by other ships at a profit.

> Where to get the resources? They likely require an infusion of outside goods.

Both of these are massively facilitated if the station is built around a [space elevator](https://en.wikipedia.org/wiki/Space_elevator) which IMHO could make a very interesting setting for a game. To your list you could also add "space tourism" -- hotels situated at different heights along the elevator can give travelers a variety of different experiences.

**Comment by kd7uiy** [comment]
The space elevator concept is, well, interesting. I don't think I'd include it for the base game, but it might be an interesting scenario to play. Hmmm...

**Comment by koosemose** [comment]
And the Orbital Space Station also has easier access to more location types, i.e. it can reach asteroids and moons more easily than a planetary base, and reach the planet (and back) more easily than an asteroid base. Obviously it has a harder time with accessing an asteroid than an asteroid base and a planet than a planetary base.

**Comment by sboigelot** [comment]
Space tourism and commerce is a very interesting revenue stream. Thinking about setup like Babylon 5 and Star Trek Deep Space 9. You could have a Value Added Tax for third party shop in your station, import and export taxes, rent, etc.

I really like the idea of a different business then "mine for ore"-"refine ores" for some space stations.

**Comment by koosemose** [comment]
Depending on the setting, an orbital base is a near necessity. It's not uncommon in Sci-Fi settings for ships that go interstellar to be incapable of landing, which means, unless they are large enough for carrying their own shuttles, having a spacebase at which they can dock (or park near and be picked up) and then get a shuttle from there down to the planet is very important, another major revenue source if one assumes at least a semi-inhabited planet

**Comment by kd7uiy** [comment]
@koosemose That's a really interesting thought. We should certainly add in something like that, although managing the docking of larger spacecraft will no doubt be tricky. Hmm...

**Comment by koosemose** [comment]
There's a limit on how large we would need to go, if they're big enough they would have their own shuttles and not need us, and that's also why I offered the idea of going to pick them up as well. so gameplay wise you send away a shuttle, and after a time they come back either having dropped the person off on planet, or bring them back to hang out for a while.

**Comment by quill18** [comment]
Science: Researching unlocks for new furniture and/or improving existing processes is always a lot of fun (see: Rimworld).  Personally, I think the actual implementation should almost feel a bit more like Kerbal Space Program: Many different types of experiments (space telescope, materials study, background radiation monitoring, etc...) that produce raw data.  Maybe each produces data slowly over time, needing only to be worked for a bit every X hours to start a new job.  Then we could either spend that data directly for unlocks, or have a full-time Science Lab (again, very Kerbally) that converts the raw data into actual science points.

Expedition: I always imagined that mining drones (and now, collecting resources from nearby asteroids) would just be the initial setup. I personally love the idea of loading up a shuttle full of supplies and sending a team of people away for X hours to mine resources. However, I'm not sure that it will actually, in the end, add that much interesting gameplay.

Repair: Oooh, I like this.  I guess we'd differentiate between the trade/tourism landing pad and the repair bay.  Dump a bunch of resources (like steel plates) and manpower into repair a ship in exchange for a bunch of money (a better conversion rate than selling the items to traders, but requires a lot of work time).

Manufacturing: Plasteel mugs, obviously.

Military: I think external threats are required to shake things up. DF and Rimworld would not be the same without invasions.  Space pirate raids should certainly be a thing -- and maybe attacks by some enemy race at some point too.

Communication: I'm not sure I see this.

Warehouse: I guess we'd be setting aside a bunch of space in exchange for passively making some money?  Not 100% sure I see the fun, but I may be misunderstanding.

**Comment by kd7uiy** [comment]
I've added some information based on Quill's feedback. The Warehouse, in my thought, would be to buy goods from one location and sell them at a profit from another location, not really a passive way to just make money, which I've included. 

I also agree that I'm not quite sure how the expedition model will work out, but hey, if someone's willing to put in the work to set it up, I don't think it would hurt anything.

I've put all of this in to the wiki, at https://github.com/TeamPorcupine/ProjectPorcupine/wiki/orbiting_base . Let me know if there's anything you think missing or poorly described, and I'll make sure it gets added. Let's leave this open for a few days to ensure that everyone is content, and then close this issue.
